src/import/FSHImporter.ts

```diff
@@ -592,6 +621,17 @@ export class FSHImporter extends FSHVisitor {
     return this.aliasAwareValue(ctx.SEQUENCE().getText());
   }
 
+  visitMixins(ctx: pc.MixinsContext): string[] {
+    if (ctx.COMMA_DELIMITED_SEQUENCES()) {
+      return ctx
+        .COMMA_DELIMITED_SEQUENCES()
+        .getText()
+        .split(/,\s*/);
+    } else {
+      return [ctx.SEQUENCE().getText()];
+    }
+  }
+
```

Comment:
This wasn't introduced in this PR, but looking at this code and the definition of `COMMA_DELIMITED_SEQUENCES`, I'm wondering if that lexer rule needs to be changed to allow whitespace between the `SEQUENCE` and the `,` tokens. The regex you're currently using is a correct fit for the current grammar, though.

Reply:
I updated the grammar and the regex to allow whitespace before the `,` in https://github.com/FHIR/sushi/pull/265/commits/7d4aa648d5ef767027064efdf0b054332a39b81a. I also updated the regex in the two other places where `COMMA_DELIMTED_SEQUENCES` are used, and updated some tests to prove that you can now have whitespace before the commas.